### ISSUE TITLE
how to generate our personnal metadata

### ISSUE THREAD

**Issue by hibaKhechai** [googleinterns/deep-stabilization#12]
Hello, I am trying to use your methode on a personal video, the FlowNet2 preparation gives me the flo files however i can't seem to understand how you generated all the informations of the texte files from the video.
is the gyro file used for that if so how can I use it ?, If not what are all the informations that i need to generate my own metadata please ?.

**Comment by cyfedu1024** [comment]
yeah, I also want to know the principle of the metadata generation. If you deal with this problem, can you share with me? my email address yfcao@mail.dlut.edu.cn. Thanks a lot!

**Comment by Lwy-1998** [comment]
Me too.  please tell me if you have any answer.
